blackjax/nuts.py

```diff
@@ -53,6 +53,7 @@ class NUTSInfo(NamedTuple):
     trajectory_leftmost_state: integrators.IntegratorState
     trajectory_rightmost_state: integrators.IntegratorState
     num_trajectory_expansions: int
+    num_leapfrogs: int
```

Comment:
Would it be possible to replace mentions of `leapfrogs` by `integration_steps`, which is more general? BlackJAX ca n be used with other integrators than leapfrog...